openlibrary/plugins/importapi/code.py

```diff
@@ -224,17 +224,37 @@ def POST(self):
         # Case 4 - Does this item have a marc record?
         marc_record = self.get_marc_record(identifier)
         if not marc_record:
+            # return self.load_book(self.get_ia_record(metadata))
```

Comment:
The import process has been changed to accept an option of requiring "marcs". When finished, if marcs=False (i.e. not required) I think we'll want to call this block.

For the interim, we can just remove it and punt on this feature.